src/client/telemetry/index.ts

```diff
@@ -844,9 +844,9 @@ export interface IEventNamePropertyMapping {
      */
     [Telemetry.KernelLauncherPerf]: undefined | never | TelemetryErrorProperties;
     /**
-     * Total time taken to find a kernel on disc.
+     * Total time taken to find a kernel on disc or on a remote machine.
      */
-    [Telemetry.KernelFinderPerf]: undefined | never;
+    [Telemetry.KernelFinderPerf]: { type: 'remote' | 'local' };
```

Comment:
Do we need this? Doesnt feel necessary, as we have telemetry tracking whether user is in local or remote.